stock_move_auto_assign_auto_release/models/product_product.py

```diff
@@ -11,7 +11,7 @@ class ProductProduct(models.Model):
     def _moves_auto_release_domain(self):
         return [
             ("product_id", "=", self.id),
-            ("is_auto_release_allowed", "=", True),
+            ("state", "not in", ("done", "cancel")),
```

Comment:
What about doing this instead ?
```suggestion
            ("need_release", "=", True),
```

Reply:
I can test that :+1:

Reply:
Done, performance are even better